packages/edit-site/src/components/sidebar-navigation-screen-pages-dataviews/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalTruncate as Truncate,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { layout } from '@wordpress/icons';
+import { useMemo } from '@wordpress/element';
+import { useEntityRecords } from '@wordpress/core-data';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useLink } from '../routes/link';
+import { TEMPLATE_POST_TYPE } from '../../utils/constants';
+import SidebarNavigationItem from '../sidebar-navigation-item';
+import SidebarNavigationScreen from '../sidebar-navigation-screen';
+import DataViewsSidebarContent from '../sidebar-dataviews';
+
+const PageItem = ( { postType = 'page', postId, ...props } ) => {
+	const linkInfo = useLink(
+		{
+			postType,
+			postId,
+		},
+		{
+			backPath: '/page',
+		}
+	);
+	return <SidebarNavigationItem { ...linkInfo } { ...props } />;
+};
+
+export default function SidebarNavigationScreenPagesDataViews() {
+	const { records: templateRecords } = useEntityRecords(
+		'postType',
+		TEMPLATE_POST_TYPE,
+		{
+			per_page: -1,
+		}
+	);
+	const templates = useMemo(
+		() =>
+			templateRecords?.filter( ( { slug } ) =>
+				[ '404', 'search' ].includes( slug )
+			),
+		[ templateRecords ]
+	);
+
+	return (
+		<SidebarNavigationScreen
+			title={ __( 'Pages' ) }
+			content={ <DataViewsSidebarContent /> }
+			footer={
+				<VStack spacing={ 0 }>
```

Comment:
Maybe it's better to extract this and reuse it?

Reply:
Where would it need reusing? Note that the existing pages root is going to be removed when the new pages dataviews is made stable, so there's only one place where we need this.

Reply:
Yeah, I see your point. I'm working to make it stable for 6.5, a few weeks from now, so I want to be optimistic. Considering that timeline, it felt unnecessary to make this PR bigger by extracting a new component that was to be removed soon. Could have done it differently, in a less optimistic day :)